source/opt/remove_duplicates_pass.cpp

```diff
@@ -151,8 +183,8 @@ bool RemoveDuplicatesPass::RemoveDuplicateDecorations() const {
 
   analysis::DecorationManager decoration_manager(context()->module());
   for (auto* i = &*context()->annotation_begin(); i;) {
-    // Is the current decoration equal to one of the decorations we have aready
-    // visited?
+    // Is the current decoration equal to one of the decorations we have
+    // aready visited?
```

Comment:
Since you are fixing the formating here, could you fix the typo as well? Should be "already visited".

Reply:
Fixed